### PR TITLE
add HarvestSkipURI

### PR DESCRIPTION
**Pull request by pdowler**
forked existing HarvestSkip class instead of using generics because it might go away entirely

### CODE REVIEW

File: caom2persistence/src/main/java/ca/nrc/cadc/caom2/harvester/state/HarvestSkip.java [hunk 1]
```diff
@@ -109,6 +110,6 @@ public UUID getSkipID()
 
     public UUID getID()
     {
-        return skipID;
+        return id;
```
Comment (brianmajor) [inline caom2persistence/src/main/java/ca/nrc/cadc/caom2/harvester/state/HarvestSkip.java]: Was this a bug before?

File: caom2persistence/src/main/java/ca/nrc/cadc/caom2/harvester/state/HarvestSkipURI.java [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+        this.source = source;
+        this.cname = cname;
+        this.skipID = skipID;
+        this.errorMessage = msg;
+    }
+
+    @Override
+    public String toString()
+    {
+        return "HarvestSkip[" + source + "," + cname + "," + skipID + "]";
```
Comment (brianmajor) [inline caom2persistence/src/main/java/ca/nrc/cadc/caom2/harvester/state/HarvestSkipURI.java]: Not sure of your intention, but you may have wanted this to print 'HarvestSkipURI'

File: caom2persistence/src/main/java/ca/nrc/cadc/caom2/harvester/state/HarvestSkip.java (hunk 1 above)
Comment (pdowler) [inline caom2persistence/src/main/java/ca/nrc/cadc/caom2/harvester/state/HarvestSkip.java]: yes, that was a bug

File: caom2persistence/src/main/java/ca/nrc/cadc/caom2/harvester/state/HarvestSkipURI.java (hunk 2 above)
Comment (pdowler) [inline caom2persistence/src/main/java/ca/nrc/cadc/caom2/harvester/state/HarvestSkipURI.java]: will change

**Pull request merged**